modules/zypper/init.zsh

```diff
@@ -0,0 +1,31 @@
+#
+# Defines zypper aliases.
+#
+# Authors:
+#   Simon <contact@saimon.org>
+#   Sorin Ionescu <sorin.ionescu@gmail.com>
+#   Andrea Ceccarelli <gltch2003@gmail.com>
+#
+
+# Return if requirements are not found.
+if (( ! $+commands[zypper] )); then
+  return 1
+fi
+
+#
+# Aliases
+#
+
+alias zypc='sudo zypper clean -a'                     # Cleans the cache.
+alias zyph='zypper history'                           # Displays history.
+alias zypi='sudo zypper install'                      # Installs package(s).
+alias zypl='zypper clean'                             # Clean packages cache.
+alias zypq='zypper info'                              # Displays package information.
+alias zypr='sudo zypper remove'                       # Removes package(s).
```

Comment:
It looks like in other packages (like `pacman`) we've used `x` rather than `r` for remove, but I feel like this is mostly personal preference. I don't have a strong opinion on this.